public/app/features/alerting/unified/components/amroutes/AmSpecificRouting.tsx

```diff
@@ -43,12 +44,18 @@ export const AmSpecificRouting: FC<AmSpecificRoutingProps> = ({
       <h5>Specific routing</h5>
       <p>Send specific alerts to chosen contact points, based on matching criteria</p>
       {!routes.receiver ? (
-        <EmptyArea
-          buttonIcon="rocket"
-          buttonLabel="Set a default contact point"
-          onButtonClick={onRootRouteEdit}
-          text="You haven't set a default contact point for the root route yet."
-        />
+        readOnly ? (
+          <EmptyArea>
+            <p>There is no default contact point configured for root route.</p>
```

Comment:
nit: not sure about the grammar
```suggestion
            <p>There is no default contact point configured for the root route.</p>
```